short-read-mngs/Dockerfile

```diff
@@ -1,3 +1,27 @@
+FROM ubuntu:18.04 AS s3quilt
+
+RUN apt-get update && apt-get install -y curl python3-pip
+
+ARG GO_VERSION=1.19.2
+RUN curl -Lo go${GO_VERSION}.linux-amd64.tar.gz https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz
+RUN tar -C . -xzf go${GO_VERSION}.linux-amd64.tar.gz && rm go${GO_VERSION}.linux-amd64.tar.gz
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:$PATH
+RUN /go/bin/go version
+WORKDIR /go/src/s3quilt
+COPY s3quilt/go.mod s3quilt/go.sum s3quilt/s3quilt.go ./
+
+RUN go get golang.org/x/tools/cmd/goimports
+RUN go install golang.org/x/tools/cmd/goimports
+RUN go get github.com/go-python/gopy
```

Comment:
looks like these aren't versioned. Might want to look into tacking in `go mod` support at some point

Reply:
fixed!